secure_message/services/internal_user_service.py

```diff
@@ -25,15 +25,19 @@ def get_user_details(uuid):
             "Authorization": "Bearer " + uaa_token.get("access_token"),
             "Content-Type": "application/json",
         }
-
+        response = requests.get(url, headers=headers)
         try:
-            response = requests.get(url, headers=headers)
             response.raise_for_status()
             resp_json = response.json()
         except HTTPError:
-            logger.exception("Failed to get user info", uuid=uuid)
             user_details = InternalUserService.get_default_user_details(uuid)
-            return user_details
+            if response.status_code == 404:
+                logger.info("Failed to get user info", uuid=uuid)
```

Comment:
Is it worth this being a more specific log message just for the 404?

Reply:
This has been updated.